Flaky test `TestCloseFlushWithTimer`

https://github.com/apache/pulsar-client-go/actions/runs/10992829338/job/30518018342?pr=1288#step:5:965

```
=== RUN   TestCloseFlushWithTimer
    ack_grouping_tracker_test.go:256: 
        	Error Trace:	/pulsar/pulsar-client-go/pulsar/ack_grouping_tracker_test.go:256
        	Error:      	Should be true
        	Test:       	TestCloseFlushWithTimer
    ack_grouping_tracker_test.go:257: 
        	Error Trace:	/pulsar/pulsar-client-go/pulsar/ack_grouping_tracker_test.go:257
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2
        	Test:       	TestCloseFlushWithTimer
--- FAIL: TestCloseFlushWithTimer (0.00s)
```

--- a/pulsar/ack_grouping_tracker_test.go
+++ b/pulsar/ack_grouping_tracker_test.go
@@ -243,7 +243,7 @@ func TestCloseFlushWithoutTimer(t *testing.T) {
 func TestCloseFlushWithTimer(t *testing.T) {
 	var acker mockAcker
 	tracker := newAckGroupingTracker(
-		&AckGroupingOptions{MaxSize: 1000, MaxTime: 10 * 1000},
+		&AckGroupingOptions{MaxSize: 1000, MaxTime: 10 * time.Second},
 		nil,
 		func(id MessageID) { acker.ackCumulative(id) },
 		func(ids []*pb.MessageIdData) { acker.ack(ids) },

